src/types.ts

```diff
@@ -73,7 +73,7 @@ export interface Service {
 	// extends?: string | { file?: string; service: string }; // unsupported for now
 	// external_links?: ListOfUniqueItems<string>; // unsupported for now
 
-	extra_hosts?: Dict<string>;
+	extra_hosts?: ListOrDict;
```

Comment:
I think in this file we define what we normalise to, so it should be `ListOfUniqueItems<string>`. (I may be mistaken, please have a look on the other types and ignore me if I am).